Resources/Prototypes/Entities/Markers/Spawners/Random/bluespace_harvester.yml

```
@@ -0,0 +1,709 @@
+# Branchs
+- type: entity
+  id: RandomHarvesterDestructionLoot
+  name: random harvester biological loot spawner
+  suffix: Bluespace Harvester, DO NOT MAP
+  parent: MarkerBase
+  noSpawn: true
+  components:
+  - type: Sprite
+    layers:
+      - state: red
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)